libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectMapper.kt

```diff
@@ -109,19 +115,20 @@ internal fun com.mapbox.navigator.RoadObject.mapToRoadObject(): RoadObject {
                 length,
                 location,
                 provider,
-                this
+                isUrban,
+                this,
             )
-        RoadObjectType.BRIDGE -> Bridge(id, length, location, provider, this)
-        RoadObjectType.CUSTOM -> Custom(id, length, location, provider, this)
+        RoadObjectType.BRIDGE -> Bridge(id, length, location, provider, isUrban, this)
+        RoadObjectType.CUSTOM -> Custom(id, length, location, provider, isUrban, this)
         RoadObjectType.RAILWAY_CROSSING -> RailwayCrossing(
             id,
             metadata.railwayCrossingInfo.toRailwayCrossingInfo(),
             length,
             location,
             provider,
-            this
+            isUrban,
+            this,
         )
-        else -> throw IllegalArgumentException("unsupported type: $type")
```

Comment:
else removed, because `RoadObjectType`(is enum) is the exhaustive structure. 

jfyi starting with kotlin `v1.7.0` compiler forces to not use `else` for exhaustive structures (sealed, enum, boolean) [ref](https://youtrack.jetbrains.com/issue/KT-12380)